### PR TITLE
Microbundle

### PR DESCRIPTION
**Pull request by olance**
Parting from the original "TypeScript lib starter" repository as it was outdated and with no obvious upgrade path.

Now using [microbundle](https://github.com/developit/microbundle) + configured commitizen/commitlint & semantic-release

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-
 ## Instructions
 
 These steps will guide you through contributing to this project:
 
 - Fork the repo
 - Clone it and install dependencies
 
-		git clone https://github.com/YOUR-USERNAME/typescript-library-starter
+		git clone https://github.com/YOUR-USERNAME/alma-js-client
```
Comment (Taskim) [inline CONTRIBUTING.md:8]: `git clone https://github.com/alma/alma-js-client` ?

File: package.json [hunk 2]
```diff
@@ -73,57 +109,38 @@
       "src/*.{js,ts}"
     ]
   },
+  "lint-staged": {
```
Comment (Taskim) [inline package.json:113]: ça peut être cool d'ajouter les tests aussi ici :)

**Review by Taskim — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (olance) [reply CONTRIBUTING.md:8]: Right! The template I had used assumed you had first forked/copied over their files to another repo!

File: package.json (hunk 2 above)
Comment (olance) [reply package.json:113]: En `pre-commit` ça me semble un peu violent 😅 
Ils sont lancés en `pre-push` 🙂

**Review by Taskim — APPROVED** [review]

**Pull request merged**